kunta-api-server/src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ServiceChannelProvider.java

```diff
@@ -0,0 +1,23 @@
+package fi.otavanopisto.kuntaapi.server.integrations;
+
+import java.util.List;
+
+import fi.otavanopisto.kuntaapi.server.rest.model.ServiceElectronicChannel;
+
+/**
+ * Provider for service channels
+ * 
+ * @author Antti Leppä
+ */
+public interface ServiceChannelProvider {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "ServiceChannelProvider" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1609)